src/index.js

```diff
@@ -425,6 +475,44 @@ function setCookiesAndTokens(tokens) {
   setTokensFromCookies();
 }
 
+/**
+ * Define user attributes & methods based on access & ID token.
+ */
+async function setUser() {
+  if (!store.idToken) {
+    throw new Error("ID token has not been set.");
+  }
+  if (!store.accessToken) {
+    throw new Error("Access token has not been set.");
+  }
+
+  try {
+    await verifyToken(store.idToken);
+  } catch (error) {
+    throw error;
+  }
+  console.log("Setting user", store);
+  store.user = user({
+    store,
+    afterUpdate: refresh,
+  });
```

Comment:
Assignment with `=` destroys any existing references to `user` that have been exported as

```js
const { user } = Userfront
```